shell/src/main/java/alluxio/cli/fsadmin/journal/QuorumElectCommand.java

```diff
@@ -60,11 +90,92 @@ public static String description() {
 
   @Override
   public int run(CommandLine cl) throws IOException {
-    JournalMasterClient jmClient = mMasterJournalMasterClient;
-    String serverAddress = cl.getOptionValue(ADDRESS_OPTION_NAME);
-    NetAddress address = QuorumCommand.stringToAddress(serverAddress);
+    NetworkAddressUtils.ServiceType serviceType =
+        NetworkAddressUtils.ServiceType.MASTER_RAFT;
+    final JournalMasterClient jmClient;
+    if (cl.hasOption(DOMAIN_OPTION_NAME)) {
+      String domainVal = cl.getOptionValue(DOMAIN_OPTION_NAME);
+      try {
+        JournalDomain domain = JournalDomain.valueOf(domainVal);
+        if (domain == JournalDomain.JOB_MASTER) {
+          jmClient = mJobMasterJournalMasterClient;
+          serviceType = NetworkAddressUtils.ServiceType.JOB_MASTER_RAFT;
+        } else {
+          jmClient = mMasterJournalMasterClient;
```

Comment:
initialize `jmClient` above to avoid `else`?

Reply:
I have to use final to limit the `jmClient` since it is used in the following lambda expression L180